G_channels.md

```diff
@@ -249,6 +249,81 @@ socket = assign(socket, :user, msg["user"])
 
 Sockets store assigned values as a map in `socket.assigns`.
 
+#### Using Token Authentication
+
+When we connect, we'll often need to authenticate the client. Fortunately, this is a fairly easy 4-step process with `Phoenix.token`. 
+
+Step 1 - Assign Token in Connection:
```

Comment:
I think we can bring back the articles here. Maybe we could drop the colon as well. That would be:

Step 1 - Assign a Token in the Connection